benches/client_mock_tcp.rs

```diff
@@ -87,6 +87,8 @@ impl net::NetworkStream for MockStream {
     fn peer_name(&mut self) -> IoResult<SocketAddr> {
         Ok(from_str("127.0.0.1:1337").unwrap())
     }
+    fn mark_dead(&mut self) {
```

Comment:
(I can't wait for `rustfmt`; good way to learn idioms is to have them thrust upon you.)